src/components/graph/menus/create-node-menu.js

```diff
@@ -255,6 +269,31 @@ const CreateNodeMenu = ({
         },
     };
 
+    const SubMenuItem = ({ item }) => {
```

Comment:
It is forbidden to create nested component in another component. Anti pattern, performance issue.
Move this component definintion in another file or outside of the CreateNodeMenu component.
Or create a render function instead.
Also `SubMenuItem` is misnamed. Something like `NodeMenuItem` is better